.github/workflows/phpcs.yml

```
@@ -0,0 +1,34 @@
+name: Code Standards
+on:
+  pull_request:
+
+jobs:
+  phpcs:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0  # Required for phpcs-changed to compare with base branch
+        
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: '8.1'
+        tools: composer:v2
+        
+    - name: Install Dependencies
+      run: composer install
+        
+    - name: Run PHPCS on changed files
+      run: |
+        CHANGED_FILES=$(git diff --name-only --diff-filter=d origin/${{ github.base_ref }} | grep ".php$" || true)
+        if [ ! -z "$CHANGED_FILES" ]; then
+          vendor/bin/phpcs-changed \
+            --git-base=origin/${{ github.base_ref }} \
+            --phpcs-path=vendor/bin/phpcs \
+            --standard=.phpcs.xml.dist \
+            $CHANGED_FILES
+        else
+          echo "No PHP files changed"
+        fi
```

It's risky to substitute GitHub variables directly into shell scripts. Safer to use environment variables.

Also, you'll want to quote those environment veriables where they're used.

You may also want to pass `--no-renames` to `git diff`, otherwise it may sometimes get confused.

No need for a grep, just tell `git diff` to only do `.php` files directly.

Then, just a pet peeve, since it's bash we can use the built-in `[[ ]]` instead of `[ ]`.
```suggestion
      env:
        BASE_REF: ${{ github.base_ref }}
      run: |
        CHANGED_FILES=$(git diff --name-only --no-renames --diff-filter=d "origin/$BASE_REF" '*.php' || true)
        if [[ -n "$CHANGED_FILES" ]]; then
          vendor/bin/phpcs-changed \
            --git-base="origin/$BASE_REF" \
            --phpcs-path=vendor/bin/phpcs \
            --standard=.phpcs.xml.dist \
            $CHANGED_FILES
        else
          echo "No PHP files changed"
        fi
```